### ISSUE TITLE
The onContentChangedByOuterProcess method under native-bridge.cpp has a JNI calling error.

### ISSUE THREAD

**Issue by wzw617248224** [Tencent/MMKV#363]
the `onContentChangedByOuterProcess` method defined at `MMKVContentChangeNotification.java` as an interface callback which **_return type is void_**

```java
public interface MMKVContentChangeNotification {

    // content change notification of other process
    // trigger by getXXX() or setXXX() or checkContentChangedByOuterProcess()
    void onContentChangedByOuterProcess(String mmapID);
}
```

but under `native-bridge.cpp`, the calling method is wrong

```c
void onContentChangedByOuterProcess(const std::string &mmapID) {
    auto currentEnv = getCurrentEnv();
    if (currentEnv && g_callbackOnContentChange) {
        jstring str = string2jstring(currentEnv, mmapID);
        
        //CallStaticIntMethod isn't the original method
        currentEnv->CallStaticIntMethod(g_cls, g_callbackOnContentChange, str);
    }
}

```

this maybe make newers misunderstanding, we just need change that into below code,
```c
void onContentChangedByOuterProcess(const std::string &mmapID) {
    auto currentEnv = getCurrentEnv();
    if (currentEnv && g_callbackOnContentChange) {
        jstring str = string2jstring(currentEnv, mmapID);
        currentEnv->CallStaticVoidMethod(g_cls, g_callbackOnContentChange, str);
    }
}
```

**Comment by lingol** [comment]
Already fixed in the `dev` branch.
Thanks anyway.

**Comment by lingol** [comment]
Released with [v1.0.24](https://github.com/Tencent/MMKV/releases/tag/v1.0.24).
